src/main/java/flaxbeard/steamcraft/block/BlockBoiler.java

```diff
@@ -35,277 +35,286 @@
 
 public class BlockBoiler extends BlockSteamTransporter implements IWrenchable {
```

Comment:
Did I spoil  the indentation of the class? Sorry about that. I must be more careful next time.